### PR TITLE
Add Esteban GitHub username to the list of students Closes #11

### LINKED ISSUE #11: Add myself (@EstebanLopezGomez) to the list of students
I would like to add myself to the list of students in the GitHub
organization README.

### PR DESCRIPTION
**Pull request by EstebanLopezGomez**

### CODE REVIEW

**Comment by EstebanLopezGomez** [comment]
Added User

**Comment by EstebanLopezGomez** [comment]
Added User @teaching-staff 

https://github.com/heig-vd-dai-course/.github/issues/11

**Comment by hadrylouis** [comment]
It would be great if you can sign your previous commits. You can follow [this tutorial](https://stackoverflow.com/a/54987693)

File: profile/README.md [hunk 1]
```diff
@@ -3,12 +3,10 @@
 Welcome to the DAI course at [HEIG-VD](https://heig-vd.ch), Switzerland!
 
 ## Getting started
-
```
Comment (ludelafo) [inline profile/README.md:6]: You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.

File: profile/README.md [hunk 2]
```diff
@@ -3,12 +3,10 @@
...
 Do not know where to start? Have a look at the
 [main repository for the course](https://github.com/heig-vd-dai-course/heig-vd-dai-course)!
 
 ## Teaching staff
-
```
Comment (ludelafo) [inline profile/README.md:11]: You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.

File: profile/README.md [hunk 3]
```diff
@@ -18,13 +16,15 @@ First name Last name [@GitHub username](https://github.com/USERNAME)
 - Hadrien Louis [@hadrylouis](https://github.com/hadrylouis)
 
 ## Students
-
```
Comment (ludelafo) [inline profile/README.md:21]: You shouldn't have removed the line between the title and the HTML comment. Maybe it's your IDE that did format the file for you? Please re-add the lines.

File: profile/README.md [hunk 4]
```diff
@@ -18,13 +16,15 @@ First name Last name [@GitHub username](https://github.com/USERNAME)
...
 <!--
 Please add your name in alphabetical order (by last name) in this format:
 First name Last name [@GitHub username](https://github.com/USERNAME)
 -->
 
+=======
+
```
Comment (ludelafo) [inline profile/README.md]: You left this from a previous conflict. You can remove these lines.

**Review by ludelafo — CHANGES_REQUESTED** [review]

**Review by ludelafo — APPROVED** [review]

**Pull request merged**